SPARCore/GADArticle.m

```diff
@@ -51,57 +89,40 @@ - (void) fetchFulltextWithCompletion: (void(^_Nonnull)(GADRemoteModel * *_Nullab
     
 }
 
-+ (NSArray <GADArticle *> *) articlesFromJSON: (NSData *)json {
++ (NSArray *) fullArticleFromJsonData: (NSData *)json {
```

Comment:
It's just because the structures of json string returned by list of articles and full article are not the same... I tried revising fetchFullTextWithCompletion into a more compact version and will push it in a new commit